src/components/DashKit/DashKit.tsx

```diff
@@ -30,9 +32,9 @@ interface DashKitDefaultProps {
     itemsStateAndParams: ItemsStateAndParams;
     settings: SettingsProps;
     context: ContextProps;
-    overlayControls?: Record<string, OverlayControlItem[]>;
-    noOverlay?: boolean;
-    draggableHandleClassName?: string;
+    noOverlay: boolean;
```

Comment:
Why did you change it from optional to required? Looks like breaking change

Reply:
I don't know who added it, but if some props are in `DashKitDefaultProps` it means that props should have values